### DISCUSSION TITLE
Please help me to achieve galaxy brain

### DISCUSSION THREAD

**Discussion by razuahmed75** [Schweinepriester/github-profile-achievements#504] [Q&A] [answered]
Please help me to achieve galaxy brain .

**Comment by fcasamento** [comment, marked as answer]
@razuahmed75 Of course, be one of the first to answer two different questions. And wait for the question owner to mark your answer as the correct one.
Here can be some places you can answer... in addition to this profile we are of course. https://github.com/STRINGLABITSOLUTIONS/Bhojon/discussions or https://github.com/orgs/community/discussions
Hope this helps.

**Comment by andrewtanjaya** [comment]
> Please help me to achieve galaxy brain .

Suree lets exchangee answers then haahaha @razuahmed75

**Comment by Preetiraj3697** [comment]
go to this disccussion https://github.com/community/community/discussions/39806 and answer them I mark your answer to help you to achieve galaxy brain.
